app/components/users/profile/projects_component.html.erb

```diff
@@ -0,0 +1,36 @@
+<%=
+  component_wrapper do
+    flex_layout(mt: 3) do |details_container|
+      details_container.with_row do
+        flex_layout(align_items: :center, justify_content: :space_between) do |heading|
+          heading.with_column(flex: 1) do
+            render(Primer::Beta::Heading.new(tag: :h4)) { t(:label_project_plural) }
+          end
+
+          if User.current.admin?
+            heading.with_column do
+              render(Primer::Beta::Link.new(
+                href: edit_user_path(@user, tab: :memberships),
+                scheme: :secondary
+              )) { t(:label_manage) }
+            end
+          end
+        end
+      end
+
+      details_container.with_row(mt: 2) do
+        render(OpPrimer::ExpandableListComponent.new(cutoff_limit: 5)) do |list|
+          @memberships.each do |membership|
+            list.with_element do
+              concat helpers.link_to_project(membership.project)
+
+              roles = membership.roles.sort.collect(&:to_s).join(', ')
+              date = helpers.format_date(membership.created_at)
+              concat render(Primer::Beta::Text.new(ml: 1, color: :subtle, font_size: :small)) { "(#{roles}, #{date})" }
```

Comment:
The figma files does not specify the listing of the roles. I however would prefer having the roles. Don't know which information need the date is catering to, though.

Reply:
The previous profile showed them, so we'll keep them as is